### PR TITLE
[ACR] Default Disable the Pull Request Trigger for Task

### PR DESCRIPTION
**Pull request by jaysterp**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by jaysterp** [comment]
//cc @northtyphoon @sajayantony @juliusl @shahzzzam

**Comment by jaysterp** [comment]
@zikalino could you please take a look? Thank you.

**Comment by jaysterp** [comment]
@yugangw-msft can you please review? We are trying to release this ASAP.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Release History
 
 **ACR**
 
+* Disable the pull request trigger by default for ACR Tasks
```
Comment (yugangw-msft) [inline src/azure-cli/HISTORY.rst]: I suggest you call out this is a breaking change by starting with `[BREAKING]`

Comment (jaysterp) [reply src/azure-cli/HISTORY.rst]: See fix.

**Pull request merged**